torch/fx/passes/utils/matcher_utils.py

```diff
@@ -217,7 +221,8 @@ def _match_args(args1: Union[List, Tuple], args2: Union[List, Tuple]) -> bool:
                 elif isinstance(a1, (list, tuple)) and isinstance(a2, (list, tuple)):
                     matched = _match_args(a1, a2)
                 else:
-                    matched = self._match_literals(a1, a2, match)
+                    if not self.ignore_literals:
```

Comment:
I see that `_match_literals` has a logic where only when a2 is not Node it returns true iff a1.op==placeholder. DOes that cover the case? What if both a1 and a2 are placeholders?

Reply:
That gets covered in line 219 `if isinstance(a1, Node) and isinstance(a2, Node)`.